spec/counter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Counter do
+  describe '.initialize' do
+    it 'creates a counter' do
+      expect(Kitchen::Counter.new).to be_truthy
+    end
+  end
+  describe '.increment' do
+    counter = Kitchen::Counter.new
```

Comment:
```ruby
RSpec.describe Kitchen::Counter do
  let(:instance) { described_class.new }
```

You can then use `instance` instead of saying `counter = Kitchen::Counter.new` below